go.mod

```diff
@@ -21,3 +21,5 @@ require (
 replace gopkg.in/urfave/cli.v1 => gopkg.in/urfave/cli.v1 v1.20.0
 
 replace github.com/ethereum/go-ethereum => github.com/vulcanize/go-ethereum v0.0.0-20190731183759-8e20673bd101
+
+replace github.com/makerdao/vulcanizedb => github.com/makerdao/vulcanizedb v0.0.14-rc.1.0.20200430184332-ecfa2869a661
```

Comment:
Once https://github.com/makerdao/vulcanizedb/pull/84 is merged, I was thinking of cutting a new VDB version and using that here.

Reply:
Not opposed to cutting a release and updating that way, but I think we could also solve this matter in the short term by running `go get github.com/makerdao/vulcanizedb@modify-unchecked-diffs-query`. Maybe nice to just modify the import directly instead of using a replace?